packages/relay/src/lib/clients/nodeClient.ts

```diff
@@ -0,0 +1,155 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+    AccountBalance,
+    AccountBalanceQuery,
+    AccountId, AccountInfoQuery,
+    Client,
+    ContractByteCodeQuery,
+    ContractCallQuery,
+    EthereumTransaction,
+    ExchangeRates,
+    FileContentsQuery,
+    ContractId,
+    ContractFunctionResult,
+    TransactionResponse,
+    AccountInfo,
+    HbarUnit
+} from '@hashgraph/sdk';
+import { BigNumber } from '@hashgraph/sdk/lib/Transfer';
+
+export class NodeClient {
```

Comment:
Is there a better name than NodeClient?  Since we are using Node.js as a framework I am very concerned about overloaded meanings of the word node.

Reply:
HederaClient is fine with me.
Yeah moved away from `ConsensusNodeClient` cause the SDK also calls the MirrorNode.
2 other options are
- SDKClient
- Add `clientMain` in `mirrroNodeClient.ts` so all mirrorNode calls come from that object and then return `nodeClient to `consensusNodeClient`

Will do `HederaClient` for now though

Reply:
went w `sdkClient`